packages/typespec-python/test/generic_mock_api_tests/test_typetest_property_optional.py

```diff
@@ -83,6 +83,22 @@ def test_int_literal(client):
     client.int_literal.put_default(models.IntLiteralProperty())
 
 
+def test_plaindate(client):
+    body = models.PlaindateProperty(property="2022-12-12")
+    assert client.plaindate.get_all() == body
+    assert client.plaindate.get_default() == models.PlaindateProperty()
+    client.plaindate.put_all(body)
+    client.plaindate.put_default(models.PlaindateProperty())
+
+
+def test_plaintime(client):
+    body = models.PlainTimeProperty(property="13:06:12")
+    assert client.plaintime.get_all() == body
+    assert client.plaintime.get_default() == models.PlainTimeProperty()
+    client.plaintime.put_all(body)
+    client.plaintime.put_default(models.PlainTimeProperty())
```

Comment:
It is better one Test case one api, so that CI could report all possible failures in one time

Reply:
update, please help rereview @msyyc